jetty-core/jetty-tests/jetty-test-client-transports/src/test/java/org/eclipse/jetty/test/client/transport/HttpClientDemandTest.java

```diff
@@ -138,20 +143,18 @@ public void process(Request request, org.eclipse.jetty.server.Response response,
         client.setResponseBufferSize(bufferSize);
         client.start();
 
-        Queue<LongConsumer> demandQueue = new ConcurrentLinkedQueue<>();
-        Queue<ByteBuffer> contentQueue = new ConcurrentLinkedQueue<>();
-        Queue<Callback> callbackQueue = new ConcurrentLinkedQueue<>();
+        Queue<Runnable> demandQueue = new ConcurrentLinkedQueue<>();
+        Queue<Content.Chunk> contentQueue = new ConcurrentLinkedQueue<>();
         CountDownLatch resultLatch = new CountDownLatch(1);
         client.newRequest(newURI(transport))
             .send(new BufferingResponseListener()
             {
                 @Override
-                public void onContent(Response response, LongConsumer demand, ByteBuffer content, Callback callback)
+                public void onContent(Response response, Content.Chunk chunk, Runnable demander)
                 {
                     // Don't demand and don't succeed callbacks.
-                    demandQueue.offer(demand);
-                    contentQueue.offer(content);
-                    callbackQueue.offer(callback);
+                    demandQueue.offer(demander);
+                    contentQueue.offer(chunk);
```

Comment:
Swap the 2 lines so they are in the same order as the parameters, and update the comment above the lines.

Reply:
Done.